modules/local/qiime2_diversity_core.nf

```diff
@@ -17,6 +17,9 @@ process QIIME2_DIVERSITY_CORE {
     path "versions.yml"                         , emit: versions
     path("*rarefaction.txt")                    , emit: depth
 
+    when:
```

Comment:
No `versions.yml` above.

Reply:
its in line 17, but I guess last line is sort of standard.